apps/condo/domains/news/schema/NewsItem.test.js

```diff
@@ -42,15 +43,14 @@ describe('NewsItems', () => {
         userClient = await makeClientWithNewRegisteredAndLoggedInUser()
         anonymousClient = await makeClient()
         sender = { dv: 1, fingerprint: faker.random.alphaNumeric(8) }
+        const [o10n] = await createTestOrganization(adminClient)
+        dummyO10n = o10n
```

Comment:
What strange names for organization?

Reply:
o r g a n i z a t i o n - is tooo looong :)
`o10n` is like i18n